examples/text-sentiment/client.py

```diff
@@ -12,16 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Standard
+import json
+
 # Third Party
 from start_runtime import protocol_arg
 import grpc
 import requests
 
 # Local
 from caikit.runtime.service_factory import ServicePackageFactory
-from text_sentiment.data_model import TextInput
+import caikit
 
 if __name__ == "__main__":
+    caikit.config.configure(
```

Comment:
For some reason I wasn't able to run the client code without this snippet. Without this, it was trying to find `sample_lib`? I'm hoping it's a config issue on my end

Reply:
@prashantgupta24 previously there was a `from text_sentiment.data_model import TextInput` bit that would import the `text_sentiment` "library" which would inject its configuration.

I like this explicit configuration better